DOCUMENTATION.md

```diff
@@ -161,4 +159,5 @@ In "Git Difference" click on the modified file to see the difference. The platfo
 ## Help and Support
 
 - Visit help section if you're facing an issue or need any help. If you have any queries or feedback, then discuss it in "Discord" or report an issue in GitHub.
-![help-and-support](https://user-images.githubusercontent.com/65342122/101987492-5efaa400-3cba-11eb-87fb-6d8cb19ae9d2.png)
+- You can check the current version of Gitconvex by clicking on the update button in this section.
+![help-and-support](https://user-images.githubusercontent.com/65342122/103559379-0b0b6580-4edc-11eb-9abe-c101f8792d42.png)
```

Comment:
@Nageswari-droid The update pane is showing an outdated version to be an available update. Please mock this so that it is not misleading